internal/provider/nordvpn/api/location.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+type location struct {
+	Country struct {
+		City struct {
+			Name string
+		}
+		Code string
+		Name string
+	}
+}
+
+func (s *server) country() string {
+	if s.Locations == nil || len(s.Locations) == 0 {
+		return ""
+	}
+	for _, location := range s.Locations {
+		return location.Country.Name
+	}
+	return ""
+}
+
+func (s *server) city() string {
+	if s.Locations == nil || len(s.Locations) == 0 {
+		return ""
+	}
+	for _, location := range s.Locations {
+		return location.Country.City.Name
+	}
+	return ""
```

Comment:
Why not just pick the first location from the server and access fields from it in the GetServers function directly, instead of using some of these methods?

I am still curious if why would a server have 0 location, or 2+ locations?